onnxruntime/python/tools/kernel_explorer/kernels/vector_add_kernel.h

```diff
@@ -41,10 +43,10 @@ __global__ void VectorAddKernel(const T* __restrict__ x,
 }
 
 template <typename T, int ThreadsPerBlock, int VecSize>
-void LaunchVectorAdd(const T* x, const T* y, T* z, int n) {
+void LaunchVectorAdd(hipStream_t stream, const T* x, const T* y, T* z, int n) {
   hipLaunchKernelGGL((VectorAddKernel<T, VecSize>), 
-                  dim3(ceil(float(n)/(float(ThreadsPerBlock)*VecSize))),
-                  dim3(ThreadsPerBlock),
-                  0, 0,
-                  x, y, z, n);
+                     dim3(ceil(static_cast<float>(n)/(static_cast<float>(ThreadsPerBlock)*VecSize))),
```

Comment:
Traditionally, I think this will be written as `((n-1) / (ThreadsPerBlock*VecSize) + 1`.
I think we need a utility function for this.

Reply:
Nice. Done.